guide/setup.md

```diff
@@ -5,14 +5,14 @@ Follow the installer on-screen instruction and proceed to launch the app. You sh
 
 ![](../images/v1.0/welcome.png)
 
-Let's now proceed to setup a Spacemesh full p2p mode, the mining process, and a Spacemesh Coins Wallet. But first, some definitions...
+Let's now proceed to setup the Spacemesh full p2p node, the mining process, and the Spacemesh Coins Wallet. But first, some definitions.
 
 ## About the Spacemesh Wallet
-> A wallet holds one or more coin accounts. An account is identified by a long unique number such as `0x20a5...3a1f`. The wallet enables you to send and receive Spacemesh Coins and in the future, will enable you run decentralized applications. Each account has a Spacemesh coins balance which you may spend.
+> A wallet holds one or more coin accounts. An account is identified by a long unique number such as `0x20a5...3a1f`. The wallet enables you to send and receive Spacemesh Coins and in the future, will enable you to run decentralized applications. Each account has a Spacemesh coin balance which you may spend.
```

Comment:
A _balance_ is not something that can be spent

Reply:
so please change to state that you can spend coins up to your account count balance.

Reply:
I meant that _coins_ are spent, not balance.

Reply:
yes, please go ahead and change that